pyupgrade.py

```diff
@@ -12,6 +12,23 @@
 import sys
 import tokenize
 import warnings
+from typing import Any
+from typing import cast
+from typing import Container
+from typing import Dict
+from typing import Iterable
+from typing import Iterator
+from typing import List
+from typing import Match
+from typing import Optional
+from typing import Pattern
+from typing import Sequence
+from typing import Set
+from typing import Text
```

Comment:
can just use `str` since I don't care about python2 mypy

Reply:
also please add `flake8-typing-imports` so I can make sure to not break people on old py35